prophy/generators.py

```diff
@@ -397,3 +401,15 @@ def setter(self, new_value):
             self._fields[field.name] = new_value
 
         setattr(cls, field.name, property(getter, setter))
+
+
+def _list_duplicates(iterable):
+    unique = set()
```

Comment:
You could use a `collections.Counter` to decrease complexity of this function. I think It could look like this:
```python
(Counter(iterable) - Counter(set(iterable))).keys()
```

Reply:
That's a nice hint. Thank you.